components/aws/sagemaker/codebuild/scripts/deploy.sh

```diff
@@ -30,6 +31,9 @@ while getopts ":d:v:" opt; do
 		v)
 			FULL_VERSION_TAG="${OPTARG}"
 			;;
+		b)
+			BUILD_VERSION="${OPTARG}"
+			;;
 	esac
 done
 
```

Comment:
docker_tag_exists function needs to be updated, not sure if public ecr allows same format, otherwise might need to use describe image/tags?

also on line 52, the version_tag is being updated according to the license file, meaning, do we have to separate the license file for v1 and v2?

Reply:
yes docker_tag_exists function needs to be updated, currently it isn't being called as the code that does so is commented out